lib/rspec/core/option_parser.rb

```diff
@@ -70,7 +70,18 @@ def parser(options)
           bisect_and_exit(argument)
         end
 
-        parser.on('--[no-]fail-fast', 'Abort the run on first failure.') do |value|
+        parser.on('--[no-]fail-fast[=COUNT]', 'Abort the run after a certain number of failures (1 by default).') do |argument|
```

Comment:
Well it should be `--profile` to be considered seperate but if that's a typo the square brackets are what should make it optional